Client/src/main/java/com/lnc/service/AuthService.java

```diff
@@ -0,0 +1,76 @@
+package com.lnc.service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.lnc.connection.ServerConnection;
+import com.lnc.controller.AdminController;
+import com.lnc.controller.ChefController;
+import com.lnc.controller.EmployeeController;
+import com.lnc.util.FromJsonConversion;
+import com.lnc.util.ToJsonConversion;
+
+public class AuthService {
+  private final ToJsonConversion jsonConversion = new ToJsonConversion();
+
+  public void authenticate(String employeeID, String password) throws JsonProcessingException {
+    String request = jsonConversion.codeLoginCredentials(employeeID, password);
+    try {
+      String response = ServerConnection.requestServer(request);
+      handleServerResponse(response, employeeID);
+    } catch (Exception ex) {
+      handleServerConnectionFailure();
+    }
+  }
+
+  private void handleServerResponse(String response, String employeeID) throws Exception {
+    switch (response) {
+      case "Wrong username or password.":
+      case "EmployeeID does not exist.":
+      case "Wrong request format.":
+        System.out.println("Wrong username or password.");
+        break;
+      default:
+        processValidResponse(response, employeeID);
```

Comment:
default should not be considered as valid response, always have a specific check for valid response